### ISSUE TITLE
congrms.tst and congsimple.tst fail when the CRISP package is loaded

### ISSUE THREAD

**Issue by wilfwilson** [gap-packages/Semigroups#376]
CRISP is a GAP package which "provides algorithms for computing subgroups of finite soluble groups". When it is loaded, the following standard test files have diffs:

* `congrms.tst`
* `congsimple.tst`

I guess this is because when CRISP is loaded, the normal subgroups of certain groups are enumerated in a different order; normal subgroups are used when calculating congruences via linked triples. Presumably, some tests in `congrms.tst`/`congsimple.tst` rely on results appearing in some particular (undocumented) order. Ideally the tests should test mathematical behaviour rather than undocumented implementational behaviour.

For example, take the following diff:
```
######## > Diff in:
tst/standard/congrms.tst:542
# Input is:
MeetSemigroupCongruences(congs[9], congs[28]);
# Expected output:
<semigroup congruence over <Rees 0-matrix semigroup 5x4 over S4>
  with linked triple (A4,4,3)>
# But found:
<semigroup congruence over <Rees 0-matrix semigroup 5x4 over S4>
  with linked triple (1,5,4)>
########
```
Rather than identifying `congs[9]` and `congs[28]` by their positions in the list `congs`, they should be identified in some mathematical way, to avoid such diffs.

**Comment by wilfwilson** [comment]
Perhaps @mtorpey could check that the diffs are harmless? Obviously, changing these test files so that they pass when CRISP is loaded is not a high priority, but I thought I'd mention it anyway.

**Comment by james-d-mitchell** [comment]
Agreed, it probably would be better to create `congs[9]`, and `congs[28]` directly from their generating pairs or otherwise, if possible.

**Comment by wilfwilson** [comment]
I resolved this with commits 6d78723 and 3ab6844.
